exchanges/bybit/bybit.go

```diff
@@ -8,740 +8,2268 @@ import (
+	}
+	if !startTime.IsZero() {
+		params.Set("startTime", strconv.FormatInt(startTime.UnixMilli(), 10))
+	}
+	if !endTime.IsZero() {
+		params.Set("endTime", strconv.FormatInt(endTime.UnixMilli(), 10))
+	}
+	if limit > 0 {
+		params.Set("limit", strconv.FormatInt(limit, 10))
+	}
+	if cursor != "" {
+		params.Set("cursor", cursor)
+	}
+	var resp *BorrowHistory
+	return resp, by.SendAuthHTTPRequestV5(ctx, exchange.RestSpot, http.MethodGet, "/v5/account/borrow-history", params, nil, &resp, defaultEPL)
+}
+
+// SetCollateralCoin decide whether the assets in the Unified account needs to be collateral coins.
+func (by *Bybit) SetCollateralCoin(ctx context.Context, coin currency.Code, collateralSwitchON bool) error {
+	params := url.Values{}
+	if !coin.IsEmpty() {
+		params.Set("coin", coin.String())
+	}
+	if collateralSwitchON {
+		params.Set("collateralSwitch", "ON")
+	} else {
+		params.Set("collateralSwitch", "OFF")
+	}
+	return by.SendAuthHTTPRequestV5(ctx, exchange.RestSpot, http.MethodGet, "/v5/account/set-collateral-switch", params, nil, &struct{}{}, defaultEPL)
+}
+
+// GetCollateralInfo retrieves the collateral information of the current unified margin account,
+// including loan interest rate, loanable amount, collateral conversion rate,
+// whether it can be mortgaged as margin, etc.
+func (by *Bybit) GetCollateralInfo(ctx context.Context, currency string) (*CollateralInfo, error) {
+	params := url.Values{}
+	if currency != "" {
+		params.Set("currency", currency)
+	}
+	var resp *CollateralInfo
+	return resp, by.SendAuthHTTPRequestV5(ctx, exchange.RestSpot, http.MethodGet, "/v5/account/collateral-info", params, nil, &resp, defaultEPL)
+}
+
+// GetCoinGreeks retrieves current account Greeks information
+func (by *Bybit) GetCoinGreeks(ctx context.Context, baseCoin string) (*CoinGreeks, error) {
+	params := url.Values{}
+	if baseCoin != "" {
+		params.Set("baseCoin", baseCoin)
+	}
+	var resp *CoinGreeks
+	return resp, by.SendAuthHTTPRequestV5(ctx, exchange.RestSpot, http.MethodGet, "/v5/asset/coin-greeks", params, nil, &resp, defaultEPL)
+}
+
+// GetFeeRate retrieves the trading fee rate.
+func (by *Bybit) GetFeeRate(ctx context.Context, category, symbol, baseCoin string) (*AccountFee, error) {
+	params := url.Values{}
+	if !common.StringDataContains(validCategory, category) {
+		return nil, fmt.Errorf("%w, valid category values are %v", errInvalidCategory, validCategory)
+	}
+	if category != "" {
+		params.Set("category", category)
+	}
+	if symbol != "" {
+		params.Set("symbol", symbol)
+	}
+	if baseCoin != "" {
+		params.Set("baseCoin", baseCoin)
+	}
+	var resp *AccountFee
+	return resp, by.SendAuthHTTPRequestV5(ctx, exchange.RestSpot, http.MethodGet, "/v5/account/fee-rate", params, nil, &resp, getAccountFeeEPL)
+}
+
+// GetAccountInfo retrieves the margin mode configuration of the account.
+// query the margin mode and the upgraded status of account
+func (by *Bybit) GetAccountInfo(ctx context.Context) (*AccountInfo, error) {
```

Comment:
Missing the field IsMasterTrader, something I desperately need against my account
https://bybit-exchange.github.io/docs/v5/account/account-info

Reply:
Me too 😭 .....